ray-operator/config/samples/tpu/tpu_list_devices.py

```diff
@@ -1,12 +1,21 @@
+import os
 import ray
 import jax
+import time
+
+from jax.experimental import multihost_utils
 
 ray.init()
 
 @ray.remote(resources={"TPU": 4})
 def tpu_cores():
-    return "TPU cores:" + str(jax.device_count())
+    cores = "TPU cores:" + str(jax.device_count())
+    print("TPU Worker: " + os.environ.get("TPU_WORKER_ID"))
+    time.sleep(10)
+    # multihost_utils.sync_global_devices("sync")
```

Comment:
forget to remove?

Reply:
Yeah I had been testing different changes in this branch and hadn't updated it yet with the correct script, I'll fix `tpu_list_devices.py` and request review again when it's ready.